cli/start.go

```diff
@@ -2,10 +2,13 @@ package cli
 
 import (
 	"fmt"
+	"net/http"
 	"time"
 
 	"golang.org/x/xerrors"
 
+	"github.com/google/uuid"
```

Comment:
this import should be grouped with line 8

Reply:
idk why CI doesn't catch this in the `fmt` job 🤔